src/Foundation.hs

```diff
@@ -154,12 +168,19 @@ instance Yesod App where
     isAuthorized RobotsR _ = return Authorized
     isAuthorized (StaticR _) _ = return Authorized
 
+    -- the profile route requires that the user is authenticated, so we
+    -- delegate to that function
     isAuthorized ProfileR _ = isAuthenticated
 
     -- This function creates static content files in the static folder
     -- and names them based on a hash of their content. This allows
     -- expiration dates to be set far in the future without worry of
     -- users receiving stale content.
+    addStaticContent
+        :: Text  -- ^ The file extension
+        -> Text -- ^ The MIME content type
+        -> Lazy.ByteString -- ^ The contents of the file
```

Comment:
It would be more idiomatic to use the existing `LByteString` type alias (inherited from classy-prelude-yesod IIRC), or to use the `BL` name that we're starting to recommend in `rio`.

Reply:
good call, I keep forgetting about what classy prelude exports!